stable/mongodb/README.md

```diff
@@ -112,10 +110,20 @@ $ helm install --name my-release -f values.yaml stable/mongodb
 
 ## Replication
 
-In order to set up a MongoDB replica set with primary, secondary and arbiter nodes you can use the values-production.yaml file.
+You can start the MongoDB chart in replica set mode with the following command:
 
 ```bash
-$ helm install --name my-release -f values-production.yaml stable/mongodb
+$ helm install --name my-release stable/mongodb --set replication.enabled=true
+```
+## Production settings and horizontal scaling
+
+The following repo contains the recommended production settings for MongoDB server in an alternative [values file](values-production.yaml). Please read carefully the comments in the values-production.yaml file to set up your environment appropriately.
```

Comment:
I would change this to,

The [values-production.yaml](values-production.yaml) file consists a configuration to deploy a scalable MongoDB deployment for production environments. We recommend that you base your production configuration on this template and adjust the parameters appropriately.

```bash
$ curl -O https://raw.githubusercontent.com/kubernetes/charts/master/stable/mongodb/production-values.yaml
$ helm install --name my-release -f values-production.yaml stable/mongodb
```

Reply:
done